css/style.css

```diff
@@ -627,6 +627,20 @@ a.btn:hover
     color: #333;
 }
 
+.btn.donate-large
+{
+    font-size: 22px;
+    padding: 0.4em 1em;
+    background-color: #D8645E;
```

Comment:
Why not `#B03931`?

Yours:

![image](https://user-images.githubusercontent.com/4370550/34682536-6cadaa4a-f49f-11e7-9387-651e19a0222c.png)

---

![image](https://user-images.githubusercontent.com/4370550/34682527-6597c31c-f49f-11e7-8387-2da2dfc184a2.png)